src/umpire/Replay.cpp

```diff
@@ -43,7 +44,7 @@ void Replay::logMessage( const std::string& message )
   if ( !replayEnabled )
     return;   /* short-circuit */
 
-  std::cout << message;
+  umpire::replay << message;
```

Comment:
@davidbeckingsale, this file actually needs to put stuff to standard output.  The output here is from the program to the user.